### DISCUSSION TITLE
How does Wave / Fusion2 work with Singularity?

### DISCUSSION THREAD

**Discussion by gwright99** [seqeralabs/wave#262] [General]
NGC is interested in leveraging Fusion2 via Wave. Their compute is an on-prem HPC which must run all containers as Singularity images. They have asked how/if Docker images pulled from via Wave are converted to Singularity images.

**Questions:**
1. Do we have existing content explaining this flow? If not:
    1. Can I please get of explanation of what is / is not possible?
    2. If possible, can someone from the Wave/Nextflow team please outline what this sequence flow looks like?

**Comment by pditommaso** [comment]
Let's start the "Discussion" section in Wave! 😄

**Reply by pditommaso** [reply]
Currently, Nextflow can convert Wave container to Singularity format as any other Docker image. However, we are looking into having Wave build natively the image in Singularity format, See [here](https://github.com/seqeralabs/wave/issues/259). 

You may want to ask if they find it useful.
